django_6pm/django_6pm/django_6pm/urls.py

```diff
@@ -15,8 +15,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import include, path
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("", include("user.urls")),
```

Comment:
It will be better if you add `account` or anyother meaningful prefix with user urls